src/main/java/org/fundacion/jala/converter/models/AssetSQL.java

```diff
@@ -72,8 +72,8 @@ public static void deleteProject(final int assetId) {
     }
 
     /**
-     * Lists the project in the db
-     * @return a list of projects
+     * Lists the project in the db.
```

Comment:
it should have a blank line between description and return.

Reply:
ok. It was solved